netcore/Microsoft.Azure.Storage.DataMovement/Microsoft.Azure.Storage.DataMovement.csproj

```diff
@@ -26,7 +26,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="..\..\lib\**\*.cs;..\..\tools\AssemblyInfo\*.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
+    <Compile Include="..\..\lib\**\*.cs;..\..\tools\AssemblyInfo\*.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**;..\..\lib\obj\**" />
```

Comment:
Please add Jason to review before official release

Reply:
Of cause, I've also sent CR to Jason.

Reply:
Discussed offline, ..\..\lib\obj\** represents temporary .cs files generated during build .net version dmlib. It is fine to exclude for this project for now.

Later, we should change the Include method to filename.cs instead of *.cs.